test/unit/src/geometries/TubeGeometry.tests.js

```diff
@@ -8,17 +8,20 @@ import {
 	TubeBufferGeometry
 } from '../../../../src/geometries/TubeGeometry';
 
+import {LineCurve3} from '../../../../src/extras/curves/LineCurve3'
+import {Vector3} from '../../../../src/math/Vector3'
+
 export default QUnit.module( 'Geometries', () => {
 
 	QUnit.module( 'TubeGeometry', ( hooks ) => {
 
 		var geometries = undefined;
 		hooks.beforeEach( function () {
 
-			const parameters = {};
+            var path = new LineCurve3(new Vector3(0, 0, 0), new Vector3(0, 1, 0));
```

Comment:
Please improve the code style in your PRs. Als ensure to use hard tabs and not soft tabs. More information here:

https://github.com/mrdoob/three.js/wiki/How-to-contribute-to-three.js